app/views/static_pages/about.html.erb

```diff
@@ -1,3 +1,11 @@
 <% provide :title, t(".about") %>
 <h1><%= t ".about" %></h1>
-<%= t "about_html" %>
+<p>
+  <%= t(".the") %> <%= link_to t(".tuto"), "https://www.railstutorial.org/" %>, <%= t(".part") %>
+  <%= link_to t(".learn"), "https://www.learnenough.com/" %> <%= t(".family") %>
+  <%= link_to t(".book"), "https://www.railstutorial.org/book" %> and
+  <%= link_to t(".screen"), "https://screencasts.railstutorial.org/" %></a>
+  <%= t(".to") %>
+  <%= link_to t(".ruby"), "https://rubyonrails.org/" %>.
+  <%= t(".this") %>
```

Comment:
thật ra máy cái này chú có thể dùng params trong I18n luôn chư sko cần tách từng biến thế này
`content: "Welcome to the %{app_name}"` và `I18n.t 'welcome', app_name: 'book store'` 
tham khảo link https://guides.rubyonrails.org/i18n.html#bulk-and-namespace-lookup